reference-implementation/to-upstream-wpts/resources/test-utils.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
This file doesn't seem to be used in this commit, so I would prefer for it to be landed in a separate commit.